### ISSUE TITLE
[Settings]: Color controls are not associated with its group label 'Terminal colors' and 'System Colors' inside Color Schemes page.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12012]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. 
4. Activate 'Color Schemes' page and then navigate to color control present below 'Terminal colors' and 'System Colors' text and observe the issue.

**User Experience:**
Screen Reader users UX will not be good if they will not be able to know the purpose of a control from its name due to lack of label association

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Color controls are not associated with its group label 'Terminal colors' and 'System Colors' inside Color Schemes page..zip](https://github.com/microsoft/terminal/files/7752159/Color.controls.are.not.associated.with.its.group.label.Terminal.colors.and.System.Colors.inside.Color.Schemes.page.zip)

### Expected Behavior

Color controls should be associated with its group label 'Terminal colors' and 'System Colors' inside Color Schemes page. 
i.e. If user navigate to 'Black' color present below 'Terminal colors' then Screen Reader should announce 'Terminal Colors Group Black' button.

### Actual Behavior

Color controls are not associated with its group label 'Terminal colors' and 'System Colors' inside Color Schemes page. 
i.e. For 'Black' color present below 'Terminal colors' then Screen Reader is only announcing 'Black' button.

**Observation:** Same issue repro with NVDA Screen Reader.

**Comment by zadjii-msft** [comment]
@PankajBhojwani (same question here)

**Comment by PankajBhojwani** [comment]
> @PankajBhojwani (same question here)

Not as of now
